### PR TITLE
Ghc911

### PR DESCRIPTION
**Pull request by yaitskov**

### CODE REVIEW

File: commutative-semigroups.cabal [hunk 1]
```diff
@@ -1,10 +1,9 @@
-cabal-version:       2.4
 name:                commutative-semigroups
 version:             0.2
 synopsis:            Commutative semigroups
 description:
   A commutative semigroup is a semigroup where the order of arguments to mappend does not matter.
-license:             BSD-3-Clause
+license:             BSD3
```
Comment (blamario) [inline commutative-semigroups.cabal]: This was no typo. See https://spdx.org/licenses/

File: commutative-semigroups.cabal [hunk 2]
```diff
@@ -1,10 +1,9 @@
-cabal-version:       2.4
```
Comment (blamario) [inline commutative-semigroups.cabal]: This field is now required, see https://cabal.readthedocs.io/en/3.6/cabal-package.html#package-properties

File: commutative-semigroups.cabal (hunk 1 above)
Comment (yaitskov) [reply commutative-semigroups.cabal]: I see. I saw build errors complaining about unknown license name, but today I wasn't able to reproduce error.

File: commutative-semigroups.cabal (hunk 2 above)
Comment (yaitskov) [reply commutative-semigroups.cabal]: reverted

**Pull request merged**